### PR TITLE
faas server and fucntion communication file format

### PR DESCRIPTION
**Pull request by sv641**
currently encode and decode for this is implemented for go.

### CODE REVIEW

File: Documentation/faas-server-to-function-communication.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Communication File format between faas server and function
+
+Information is passed between kontain faas server and function by using \<function\>.request and \<function\>.response.
+Request and response files are located in /kontain directory.
+
+Request and response files are in plain text. All information is passed as \<key\>: \<value\> pairs. key is always plain text.
+
```
Comment (paulpopelka) [inline Documentation/faas-server-to-function-communication.md:7]: What is the delimiter between key:value pairs?
Newline?

Comment (sv641) [reply Documentation/faas-server-to-function-communication.md:7]: It is right there :

Comment (paulpopelka) [reply Documentation/faas-server-to-function-communication.md:7]: That's the delimiter between the key and the value.

What separates one key:value pair from another?

Comment (sv641) [reply Documentation/faas-server-to-function-communication.md:7]: newline. One KP per line.

Comment (sv641) [reply Documentation/faas-server-to-function-communication.md:7]: mis-understood your question.

**Review by johnmuth81 — APPROVED** [review]
LGTM

**Pull request merged**